bootstrap.sh

```diff
@@ -55,6 +55,18 @@ program_exists() {
     fi
 }
 
+nvim_exists() {
+    local ret='0'
+    type 'nvim' >/dev/null 2>&1 || { local ret='1'; }
+
+    # throw error on non-zero return value
+    if [ ! "$ret" -eq '0' ]; then
+        return 1
+    fi
+
+    return 0
+}
+
```

Comment:
As @mkwmms mentioned, program_exists will exit the script if neovim is not found.